src/js/slider/slider.js

```diff
@@ -268,7 +268,7 @@ class Slider extends Component {
    *         percentage filled that the slider is
    */
   getProgress() {
-    return clamp(this.getPercent(), 0, 1).toFixed(4);
+    return Number(clamp(this.getPercent(), 0, 1).toFixed(4));
```

Comment:
`toFixed` turns the number that we get back from clamp into a `string` so 0 becomes `"0.0000"` and `0.8777657` becomes `"0.8777"` when it should be a number still.